### DISCUSSION TITLE
2006/06/19/%E6%88%91%E5%92%8C-labview-%E5%BA%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#86] [Announcements]
# 2006/06/19/%E6%88%91%E5%92%8C-labview-%E5%BA%8F

当我开始在键盘上敲打出这句话的时候，我已经使用 LabVIEW 7 年了。7 年的时间，就算天赋平平也可以积攒下一箩筐可供参考的经验了。所以我打算利用今后的闲暇时间写一些这方面的东西，既可以同大家交流，也是作为自己这七年工作的总结。

https://blog.qizhen.xyz/2006/06/19/%E6%88%91%E5%92%8C-labview-%E5%BA%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

wangfei
2011年04月11日 08:23 编辑
今天买了您的书，学习了哦，支持下！

回复
中梁
2009年09月22日 10:28 编辑
前天收到 阮大侠的书了，现在正在加紧学习，感觉这书对我们这些自学者来说，是真正能够能够带来很大提高的宝典。谢谢阮大侠了！！

回复
Zhu
2009年01月4日 10:10 编辑
我们公司也常用labview作开发，,LV的确很出色，欢迎交流。rocky.zhu@jinhorn.com

回复
伍德琳夫人的先生
2007年09月14日 03:40 编辑
你好，看了 你写的关于labview的东西 感觉帮助很大。

回复
Qizhen
2007年08月12日 14:16 编辑
一年的 LabVIEW 写作成绩今天回头查看了一下自己到底写了多少东西。杂七杂八的随笔不算，我把《我和LabVIEW》系列的文章都拷贝到了word文档里。总共有大约45篇文章，5万多字。加上图片也有一百多页纸了。和我一年前的估计差不多。说明我还是比较勤劳的:)

回复
树袋熊
2007年07月14日 11:37 编辑
我是一个初学者，有这样好的交流平台太好了!

回复
Ryan
2007年06月25日 19:34 编辑
我应为去年在接触LV,有很多的地方不是很懂得,我现在千方百计的学习LV,自己解决一写问题,实在解决不了的,能不能向高手请教??

回复
善在琳心
2007年04月23日 11:42 编辑
请问在哪里可以找到NI自己编写的 LabVIEW 中高级教程呢？我也想看看有关这方面的教程，可是还没有找到，不知可否提供一些资源，谢谢！！

回复
Unknown
2007年03月5日 11:30 编辑
谢谢你的文章，给我们初学者带来不少启发！好人呀

回复
晶
2006年11月13日 20:16 编辑
收获啊！以后打算常来看看:)

回复
Jacky
2006年10月18日 14:41 编辑
8错,以后会长来看看这里,游泳的东西顶顶顶….

回复
Qizhen
2006年08月25日 15:08 编辑
不知提供下载 LabVIEW 有否涉嫌违法，所以为了安全，我是不会在自己的 blog 里散布这些信息的。

回复
程
2006年08月25日 10:54 编辑
最近也在研究Labview，不知道什么地方可以下载到8.2啊~

回复
竹贤
2006年06月30日 19:14 编辑
清继续。

回复
竹贤
2006年06月20日 18:49 编辑
Very good!

回复
Qizhen
2006年06月20日 18:03 编辑
因为我用的是拼音输入法，待我一一把他们改正 :)

回复
Xiaohong
2006年06月20日 16:38 编辑
写的还不错，就是错别字太多了，为什么每篇文章里都要出现错别字呢？不是个好习惯

回复
76
2006年06月20日 01:11 编辑
我来咧。。。
瓦嘎嘎嘎。。。
看球ing。。。
 

回复
